client/js/scoreboard.js

```diff
@@ -33,14 +33,14 @@ export default class Scoreboard {
 	}
 
 	updateScores(players) {
+		players.sort(function(p1, p2){
+			return p2.score - p1.score;
+		})
+
+		$("#player-list li").remove();
```

Comment:
You already have the container in a property, don't call `$`.
Also, make it an `<ol>`
Also, look at https://api.jquery.com/empty/